src/bandit-evaluator.ts

```diff
@@ -23,6 +23,45 @@ export class BanditEvaluator {
   private readonly assignmentShards = BANDIT_ASSIGNMENT_SHARDS; // We just hard code this for now
   private readonly sharder: Sharder = new MD5Sharder();
 
+  public evaluateBestBandit(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: ContextAttributes,
+    actions: Record<string, ContextAttributes>,
+    banditModel: BanditModelData,
+  ): BanditEvaluation {
+    const actionScores: Record<string, number> = this.scoreActions(
+      subjectAttributes,
+      actions,
+      banditModel,
+    );
+
+    const actionWeights: Record<string, number> = this.weighActions(
+      actionScores,
+      banditModel.gamma,
+      banditModel.actionProbabilityFloor,
+    );
+
+    const selectedActionKey = Object.keys(actionScores).reduce((prev, cur) => {
+      if (!prev || actionScores[cur] > actionScores[prev]) {
+        return cur;
+      }
```

Comment:
We should break ties with action names like we do with `selectAction()` in the normal flow.

Reply:
good catch. done